### ISSUE TITLE
new mutation for octopodes: reduced damage from blunt attacks

### ISSUE THREAD

**Issue by jeremygurr** [jeremygurr/dcssca#463]
Bloax suggested this: for those with gelatinous body mutation, it makes sense that blunt attacks would do less damage, and it would be a nice situational buff for octopodes. Early game it helps a lot against ogres. Yet it does nothing for slicing attacks. I think this could add some interest to the game.

**Comment by yrmvgh** [comment]
Does the game not have enough damage types? How clear and predictable is it whether a morningstar is gonna deal blunt, spiky, or walloping damage? A flail? A tentacie? An angry sheep? A thrown sheep? Trying to remember "which kind of physical damage is it" sounds like a return to gaming's dark ages, not a leap forward.

We could take that idea of "rubbery" and have a small independent chance to halve the damage of a meiee attack? Like rMsl, but against meiee?

**Comment by jeremygurr** [comment]
I thought we already had different damage types in the game? We have crushing, slashing, piercing, chopping. Except for fighting hydras, these distinctions are meaningless. I'm not sure it's a bad thing to have a little more distinction like this between weapons. We have the same kind of distinction with magic: fire, ice, poison, etc, damage. Knowing what a creature is vulnerable to is an important part of game choice and strategy. The same goes for the player. We have fire resistance, cold resistance, etc. Why not bashing resistance? 

Just reducing all melee damage is less interesting I think.

**Comment by sooheon** [comment]
I agree with this, that the distinction is nice. I also agree with the concern about what to do about unclear cases, like thrown sheep. Still, worst case those can just remain "generic damage", and we just add interesting cases like hydra heads and gelatinous bodies for clear cut sources of damage. I can also picture doing something like making morningstars half piercing half crushing damage. As long as this is readable on the inspection screen I don't see that it'd be a huge drawback.
